packages/pds/src/app-view/services/indexing/plugins/post.ts

```diff
@@ -93,7 +93,7 @@ const insertFn = async (
       uri: external.uri,
       title: external.title,
       description: external.description,
-      thumbCid: external.thumb?.cid || null,
+      thumbCid: external.thumb?.ref?.toString() || null,
```

Comment:
I thiiink that if  `thumb` exists then it's a `BlobRef` and you're guaranteed to have a `ref`.

Reply:
Oh do you not have to keep using `?`?

I can just do `external.thumb?.ref.toString()`?

Reply:
didn't know that's how the language feature worked 👀

Reply:
Ah right on!  Yeah, I find it's kind nice since it allows you to be precise about where there may be a missing value :) and even more so with typescript!